setup.sh

```diff
@@ -11,6 +11,12 @@ wkdir=`pwd` # working directory
 bin=$wkdir/bin
 [ ! -d $bin ] && { mkdir $bin; }
 
+pm=$HOME/perl5 # perl modules config
```

Comment:
BioPerl inside VEP automatically makes the $HOME/perl directory. 
All attempts (making perl_module directory within rainbow directory, making per5 module within rainbow directory) without creating perl5 directory in $HOME failed.